src/background.ts

```diff
@@ -177,7 +186,10 @@ async function handleMessage(p: Runtime.Port, message: any) {
             },
             "body": JSON.stringify(query)
           }).then(async (resp) => {
+              browser.windows.remove(progressWindow.id!)
               if (resp.ok) {
+                let completionAudio = new Audio(chrome.runtime.getURL("earcon_server_communication_IMAGE_results-arrived.mp3")); //TODO: replace it to the audio that indicate the completion of the request
```

Comment:
Remove the comment as well, as this is the correct file and should not be replaced